example/http/client/sync/http_client_sync_post_json.cpp

```diff
@@ -0,0 +1,111 @@
+//
+// Copyright (c) 2021-2022 Guilherme Schvarcz Franco (guilhermefrancosi at gmail dot com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/boostorg/beast
+//
+
+//------------------------------------------------------------------------------
+//
+// Example: HTTP client, synchronous, JSON message
+//
+//------------------------------------------------------------------------------
+
+//[example_http_client
+
+#include <boost/asio/connect.hpp>
+#include <boost/asio/ip/tcp.hpp>
+#include <boost/beast/core.hpp>
+#include <boost/beast/http.hpp>
+#include <boost/beast/version.hpp>
+#include <boost/json/src.hpp>
```

Comment:
hmm idk about this, the CML / Jamfile should link to the actual library. we should not include src.hpp like this. And it can confuse automation tools which try to figure out library dependencies.